frontend/js/index.js

```diff
@@ -29,7 +29,7 @@ import backLinks from "./components/back-links";
 import parallax from "./components/parallax";
 import scrollWay from "./components/scroll-way";
 import articleList from "./components/article-list";
-import facebookVideo from "./components/facebook-video";
+// import facebookVideo from "./components/facebook-video";
```

Comment:
If a code is useless, then remove it. 
Or add a comment about when to un-comment this code.